test/test-utils.js

```diff
@@ -0,0 +1,13 @@
+const TokenOWL = artifacts.require("TokenOWL")
+const { toETH } = require("../scripts/utils/internals")
```

Comment:
In another PR I removed this function, so one of us would have to take care of a conflict.